_pages/gabrielvalandrobetha.yaml

```diff
@@ -0,0 +1,5 @@
+ ---
+ githubHandle:gabrielvalandrobetha
+ pageUrl:gabrielvalandrobetha.github.io/hello-world/
+ timestamp:2017-07-24
```

Comment:
@gabrielvalandrobetha ensure you're using the template below, including indentations and spaces:
```yaml
---
githubHandle: YOUR-USERNAME
pageUrl: YOUR-HANDLE.github.io
timestamp: TODAYS-DATE-YYYY-MM-DD
---
```